portfolio/src/main/webapp/script.js

```diff
@@ -35,3 +35,35 @@ function getCommentsUsingArrowFunctions() {
       });
   });
 }
+
+function createMap() {
+  const locations = [
+      ['Hagia Sophia', 41.008469, 28.980261],
+      ['Stan the T-Rex', 37.421903, -122.084674],
+      ['London', 51.503323, -0.119543],
+      ['Chandigarh', 30.733315, 76.779419],
+      ['Liandudno', 53.326540, -3.834460]
+    ];
+  const map = new google.maps.Map(document.querySelector('#map'), {
+    zoom: 1.5,
+    center: new google.maps.LatLng(0.00000, 0.00000),
+    mapTypeId: google.maps.MapTypeId.TERRAIN
+  });
+
+  const infowindow = new google.maps.InfoWindow();
+
+  var marker, i;
+  
+  for (i = 0; i < locations.length; i++) {  
+    marker = new google.maps.Marker({
+      position: new google.maps.LatLng(locations[i][1], locations[i][2]),
+      map: map
+      });
+    google.maps.event.addListener(marker, 'click', (function(marker, i) {
+      return function() {
+      infowindow.setContent(locations[i][0]);
+      infowindow.open(map, marker);
+      }
+      })(marker, i));
+    }
+ }
```

Comment:
The spacing is off here in a few places. [Style Guide link](https://google.github.io/styleguide/jsguide.html#formatting-block-indentation)
```suggestion
  for (i = 0; i < locations.length; i++) {  
    marker = new google.maps.Marker({
      position: new google.maps.LatLng(locations[i][1], locations[i][2]),
      map: map
    });
    google.maps.event.addListener(marker, 'click', (function(marker, i) {
      return function() {
        infowindow.setContent(locations[i][0]);
        infowindow.open(map, marker);
      }
    })(marker, i));
  }
}
```